appcircle_audit_permission_changes/1.0.0/component.yaml

```diff
@@ -4,7 +4,7 @@ displayName: Audit Permission Changes
 description: "Permission Differance Checker"
 webUrl: https://github.com/appcircleio/appcircle-ios-permission-check-component
```

Comment:
4th Review:

- [ ] Can you define following variables are global, because they are using in 49-51 and 90-91:
  ```
  project_path = (Pathname.new $ac_repository_path).join(Pathname.new($project_path))
    project_directory = File.dirname(project_path)
  ```

- [ ] Can you define following variable is global, because it is using in 226 and 264:
  ```
  signed_url_api = "#{$ac_callback_url}?action=getCacheUrls"
  ```

- [ ] Can you change following name, because includes must be use multi variables:
  ```
  ac_cache_included_path => ac_permission_cache_path (or like that)
  ```